utils/download_audio.sh

```diff
@@ -1,36 +1,56 @@
 #!/usr/bin/env bash
 # Copyright 2021  Jiayu Du
 #                 Seasalt AI, Inc (Author: Guoguo Chen)
-
+#                 Tsinghua University (Author: Shuzhou Chai) 
 
 set -e
 set -o pipefail
 
-if [ $# -ne 1 ]; then
-  echo "Usage: $0 <gigaspeech-dataset-dir>"
-  echo " e.g.: $0 /disk1/audio_data/gigaspeech"
+
+subset=XL
+
+. ./utils/parse_options.sh || exit 1;
+
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 [options] <gigaspeech-dataset-dir> <GigaSpeech-json> "
```

Comment:
This adds a new argument to the script, and will likely break the existing code (e.g., the Kaldi or ESPnet data download part?)

We should try to make this backward compatible, e.g., by making gigaspeech-json an optional parameter, through --gigaspeech-meta, and if the user does not provide that, we assume it's under `gigaspeech-dataset-dir`?

Reply:
And I think @chaisz19 has forgotten to add the corresponding argument in `giga_speech_download.sh`.

Reply:
Yes, that's why was saying that it was not backward compatible... I'd prefer we provide that through something like `--gigaspeech-meta` or `gigaspeech-json-meta`